EpikCord/client/client.py

```diff
@@ -91,6 +91,9 @@ async def close(self):
         self.rate_limiter.reset.cancel()
 
     async def change_presence(self, presence: Presence):
+        if not self.ws:
+            raise RuntimeError("Client is not connected to the gateway.")
```

Comment:
Wonder if it couldn't be a thing to delay it, if the client isn't connected... just an idea